### PR TITLE
Specify chart definitions in JSON

### PR DESCRIPTION
**Pull request by theosanderson**
Well I think this is kind of cool. Breaking changes re query parameters used for stacked/unstacked, but I doubt many people have bookmarks yet.

### CODE REVIEW

File: src/components/LocalIncidence.jsx [hunk 1]
```diff
@@ -75,60 +20,25 @@ function LocalIncidence ({ values, date, setDate, className, isMobile = false, l
...
-            width={width}
-            isMobile={isMobile}
-            heading='R'
-            area_data={area_data}
-            date={date}
-            setDate={setDate}
-            parameter='R'
-            stroke={lineColor}
-          />
+        <div ref={measureRef} className={classNames(`grid grid-rows-${numCharts} gap-2`, className)}>
```
Comment (richardgoater) [inline src/components/LocalIncidence.jsx:23]: There's a small tech issue here, dynamic classes don't go well with Tailwind ([link](https://tailwindcss.com/docs/optimizing-for-production#writing-purgeable-html)). If you inspect the element in the deployment, the `grid-rows-*` class is not in the bundle. Luckily it looks like the default behaviour of a grid is to split into even rows, so I think `grid-rows-*` can be removed - unless you know a case where it's not working?

**Review by richardgoater — COMMENTED** [review]
Nice one dude, a very natural progression. There's just one issue to check but otherwise it looks great.

File: src/components/LocalIncidence.jsx (hunk 1 above)
Comment (richardgoater) [reply src/components/LocalIncidence.jsx:23]: p.s. these classes might help as they don't depend on specific numbers: https://tailwindcss.com/docs/grid-auto-rows

Comment (theosanderson) [reply src/components/LocalIncidence.jsx:23]: Oh yeah - sorry that's obvious in hindsight :) Thanks for the catch!

**Pull request merged**